numba/tests/doc_examples/test_structref_usage.py

```diff
@@ -67,7 +67,7 @@ def MyStruct_get_vector(self):
 
 
 # This associates the proxy with MyStructType for the given set of
-# fields. Notice how we are not contraining the type of each field.
+# fields. Notice how we are not constraining the type of each field.
```

Comment:
Is this correct or should it be `containing` ?

Reply:
That's why I veered towards `constraining` but wasn't sure.

Reply:
Given the context, pretty sure it's `constraining`, thanks for checking.